### PR TITLE
Live operator updates

### PR DESCRIPTION
**Pull request by cquammen**
Progress towards live updates of intermediate results from long running operators, such as reconstruction operators.

### CODE REVIEW

File: tomviz/OperatorPython.cxx [hunk 1]
```diff
@@ -118,6 +120,11 @@ void OperatorPython::registerCustomWidget(const QString& key, bool needsData,
                          QPair<bool, CustomWidgetFunction>(needsData, func));
 }
 
+void OperatorPython::dataUpdated()
+{
+  dataSource()->dataChanged();
+}
+
```
Comment (cjh1) [inline tomviz/OperatorPython.cxx]: Is this used?

**Comment by cjh1** [comment]
@cquammen I have fixed up the CI issues and moved the test into a separate operator. If you are happy we can go ahead and merge so @Hovden, @yijiang1 and @jtschwar can play with it.

File: tomviz/OperatorPython.cxx (hunk 1 above)
Comment (cquammen) [reply tomviz/OperatorPython.cxx]: Nope. False start, will remove.

**Comment by cryos** [comment]
Just circling back on this - let's see if we can get it cleaned up in the next few days so that we can get it merged.

**Comment by cquammen** [comment]
Note that I've force pushed on this branch, so you'll want to reset your local branches.

I have a couple more minor cleanups to do and then I think this is ready for official review.

**Comment by cryos** [comment]
There are some clang-format/flake8 issues (not sure if it is because the fixes I pushed were lost in the force push, or new issues introduced, but it would be good to take care of them.

**Comment by cquammen** [comment]
I've done all the cleanup I wanted to do. Please review.

File: tomviz/OperatorPython.cxx (hunk 1 above)
Comment (cjh1) [reply tomviz/OperatorPython.cxx]: Shall we remove this?

Comment (cquammen) [reply tomviz/OperatorPython.cxx]: Hmm, thought I removed this. I will really do that this time :-)

**Comment by cquammen** [comment]
Okay, ready for real this time.

**Pull request merged**